### ISSUE TITLE
How to deal with browsers (usability)

### ISSUE THREAD

**Issue by babeal** [naveednajam/Little-Snitch---Rule-Groups#23]
I've added your rule set sb_unified_hosts_base.lsrules and I have another rule for chrome to allow all connections.  What's happening on almost every web page I go to is that little snitch pops up and says "For ipaddress X, there are conflicting rules".  This is because the resolved host names for that IP match both rules.  What's the best way to deal with this?

**Comment by naveednajam** [comment]
You may raise the priority of either ruleset to take priority over the other to resolve conflict.

**Comment by babeal** [comment]
How?  If i go to the rule groups and right click I don't get the option.  Select individual rules in the rule set and right click I don't get the option.  However, if i go to individual rules I created and I right click I get the option to "increase the priority".  So at this point it seems I can only increase the priority of the "allow all connections to chrome rule" and not increase the priority of the deny list

**Comment by barretto94** [comment]
I agree with @babeal - I don't see how to increase the priority of the deny rules in the Rule Group over the allow all connections rule for the Chrome process.
